.dev/tests/cypress/support/commands.js

```diff
@@ -1,8 +1,10 @@
 import { disableGutenbergFeatures, loginToSite } from '../helpers';
 
 before( function() {
-	loginToSite();
-	disableGutenbergFeatures();
+	loginToSite().then( () => {
+		cy.wait( 10000 );
```

Comment:
I do not love to see an arbitrary wait here. Do you know what we are waiting for exactly here? Perhaps we can instead assert something expected that only shows after this action is complete?

Reply:
I also do not love it ;) , but I've tried to wait for a lot of things, and there is always an error somehow. Never figured out the right thing to wait on. So yeah, I know it's suboptimal, but I would merge it for now (as release of WP 6.1 is today) and try to find a better way to do it after release.

Reply:
Sounds good to me. Should we add a quick comment there perhaps so we can easily spot it? Otherwise, we can make an effort in the future to investigate all uses of `wait` and fix it then.

Reply:
Yeah, will add a comment, good idea.

Reply:
This is so hard to do and we've gone through many solutions that failed in the past. Cypress has some weird ways to handle different paths like this but again, it's inconsistent.

I'm not sure where we're at with moving towards Storybook so it might not be worth going any further than the arbitrary wait. The change we can make knowing a wait is in our tests now is to make sure we're not running the `loginToSite` code on each test like we did in the past. There's an old PR I closed recently where I was [optimizing our tests](https://github.com/godaddy-wordpress/coblocks/pull/1319) for this reason.

Reply:
Yeah, the fact that now we are heavily parallelizing the tests is that it is adding basically 10s to the overall build time instead of 10s for each test. So, barely noticeable ;)